vendors/st/boards/stm32l475_discovery/aws_tests/application_code/main.c

```diff
@@ -503,7 +501,7 @@ void vApplicationIdleHook( void )
 
     if( ( xTimeNow - xLastPrint ) > xPrintFrequency )
     {
-        configPRINT( "." );
+        //configPRINT( "." );
```

Comment:
I think we should fix this before checking this in

Reply:
Thanks for catching that. Fixed.